src/main/java/com/github/cameltooling/lsp/internal/parser/ParserFileHelperFactory.java

```diff
@@ -20,6 +20,10 @@
 
 public class ParserFileHelperFactory {
 	
+	private static final String CAMELK_GROOVY_FILENAME_SUFFIX = ".camelk.groovy";
```

Comment:
hasn't that suffix suggestion been declined in favor of the modeline?

Reply:
it has not been declined completely.
They asked for the modeline but we don't reach an agreement on what is the best.
The proposal is to support 3 ways:
- modeline-like
- filename *.camelk.groovy
- shebang (this one will require more work on runtime side)